session/tidb_test.go

```diff
@@ -141,26 +141,7 @@ func removeStore(c *C, dbPath string) {
 
 func exec(se Session, sql string, args ...interface{}) (sqlexec.RecordSet, error) {
```

Comment:
Let us eliminate this function as it is only used in `mustExec`.

Reply:
I will take a look in a followup PR. For now it looks like we are not getting into Execute/ExecuteInternal in the tests.